cert/cert.go

```diff
@@ -271,11 +281,11 @@ func ipsToStrings(ips []net.IP) []string {
 }
 
 // IsCertExpired checks if the certificate about to expire
-func IsCertExpired(cert *x509.Certificate) bool {
+func IsCertExpired(cert *x509.Certificate, days int) bool {
```

Comment:
IMO, this should be a new function called `IsCertExpiring(cert,days)`

Reply:
I agree, it probably should have been.